dali/operators/reader/coco_reader_op.cc

```diff
@@ -20,62 +20,72 @@ DALI_REGISTER_OPERATOR(COCOReader, COCOReader, CPU);
 DALI_SCHEMA(COCOReader)
   .NumInput(0)
   .NumOutput(3)
-  .DocStr(R"code(Read data from a COCO dataset composed of directory with images
-and an annotation files. For each image, with `m` bboxes, returns its bboxes as `(m,4)`
-Tensor (``m * [x, y, w, h]`` or ``m * [left, top, right, bottom]``) and labels as `(m,1)` Tensor (``m * category_id``).)code")
+  .DocStr(R"code(Reads data from a COCO dataset that is composed of a directory with
+images and annotation files. For each image with m bboxes, the bboxes are returned as  ``(m,4)``
+Tensor (``m * [x, y, w, h]`` or ``m * [left, top, right, bottom]``) and labels as ``(m,1)``
+Tensor (``m * category_id``).)code")
   .AddOptionalArg(
     "meta_files_path",
-    "Path to directory with meta files containing preprocessed COCO annotations.",
+    "Path to the directory with meta files that contain preprocessed COCO annotations.",
     std::string())
   .AddOptionalArg("annotations_file",
       R"code(List of paths to the JSON annotations files.)code",
       std::string())
   .AddOptionalArg("shuffle_after_epoch",
-      R"code(If true, reader shuffles whole dataset after each epoch.)code",
+      R"code(If set to True, the reader shuffles the entire  dataset after each epoch.)code",
```

Comment:
Again `if set` that I'm not sure what to think about.

Reply:
Should I change it or leave it as it is?

Reply:
It's just a little more verbose. Let's leave it as is to avoid prolonguing this PR review more than we should

Reply:
I think Doxygen file follows this schema:
```
If set to YES ...
```
```
If set to NO...
```